packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -30,28 +53,72 @@ storiesOf('Components|SAYT', module)
       `
     } 
   })
-  .add('Responding to Events - sayt_show & sayt_hide ', () => {
-    setTimeout(() => {
-      window.dispatchEvent(autocompleteDataReceivedEvent);
-    }, 100);
-
-    setTimeout(() => {
-      window.dispatchEvent(new Event('sfx::sayt_show'));
-    }, 3000);
-
-    setTimeout(() => {
-      window.dispatchEvent(new Event('sfx::sayt_hide'));
-    }, 6000);
+  // @TODO Remove these setTimeouts when opening a new story
+  .add('Responding to Events - sayt_hide & sayt_show ', () => {
+    emitEventInFuture(autocompleteDataReceivedEvent, 100);
+    emitEventInFuture(new Event('sfx::sayt_hide'), 2000);
+    emitEventInFuture(new Event('sfx::sayt_show'), 4000);
 
+    const sayt = getSayt('', false);
     return `
-      <sfx-sayt></sfx-sayt>
+      ${sayt}
+      ${getDisplayCode(sayt)}
     `;
   }, {
     notes: {
       markdown:`
         # Search As You Type (SAYT)
-        - Receiving sayt_show event after 3 seconds.
-        - Receiving sayt_hide event after 6 seconds.
+        - Show automatically once sub-component Autocomplete receives results.
+        - Receiving sayt_hide event after 2 seconds.
+        - Receiving sayt_show event after 4 seconds.
       `
     }
   })
+  .add('SAYT with simple search input', () => {
+    emitEventInFuture(autocompleteDataReceivedEvent, 100);
+
+    const input = `<input type="text" id="search-bar" placeholder="Search here" />`;
+    const sayt = getSayt();
+    return `
+      ${input}
```

Comment:
indentation, spacing with {} and semi colons from here to line 122?

Reply:
I have to be careful with indentation because it can cause the displayed lines of code to be mis-indented.